src/process/drydep/CCPr_DryDep_Mod.F90

```diff
@@ -200,57 +200,59 @@ SUBROUTINE CCPr_DryDep_Run( MetState, DiagState, DryDepState, ChemState, RC )
                !    pblh, hflux, z0h, u10m, v10m, fraclake, gwettop, rc)
 
          ! loop through aerosol species
-         do i = 1, ChemState%nSpeciesAeroDryDep
-             
+         do i = 1, ChemState%nSpeciesAeroDryDep  
+
+            if (DryDepState%Resuspension) then
+
             radius = ChemState%chemSpecies(ChemState%DryDepIndex(i))%radius
             rhop = ChemState%chemSpecies(ChemState%DryDepIndex(i))%density
 
-            call CCPr_Scheme_GOCART_DryDep( MetState%NLEVS,
-                                            MetState%T,
-                                            MetState%AIRDEN,
-                                            MetState%ZMID,
-                                            MetState%LWI,
-                                            MetState%USTAR,
-                                            MetSTate%PBLH,
-                                            MetState%HFLUX,
-                                            VON_KARMAN,
-                                            Cp,
-                                            g0,
-                                            radius, 
-                                            rhop
-                                            MetState%U10M,
-                                            MetSTate%V10M,
-                                            MetState%FRACLAKE,
-                                            MetState%GWETTO
-                                            drydepf,
-                                            RC, 
-                                            DryDepState%Resuspension,
-                                            )
+            call CCPr_Scheme_GOCART_DryDep( MetState%NLEVS,   &
```

Comment:
I would suggest keeping this the way it was rather than inside the driver and making the other variables optional.  Let the scheme decide inside based on the passed DryDepState%Resuspension variable.

Reply:
done.